interface-definitions/protocols-ospf.xml.in

```diff
@@ -713,7 +713,7 @@
                 <description>Interface to be passive (i.e. suppress routing updates)</description>
               </valueHelp>
               <constraint>
-                <regex>^(br|bond|dum|en|eth|gnv|peth|tun|vti|vxlan|wg|wlan)[0-9]+|lo$</regex>
+                <regex>^(br|bond|dum|en|eth|gnv|peth|tun|vti|vxlan|wg|wlan|VLINK)[0-9]+|lo$</regex>
```

Comment:
Thanks @srividya0208 ,

with commit https://github.com/vyos/vyos-1x/commit/96d51fa69fbe072fe5c9e8efd4fa6a89d8a2771a the copied over and over - but still incomplete - regex was moved to a new interface-name general verifyer. Please rebase you work and incorporate the following changes:
- Add vlink as completionn helper
- Describe vlink via valueHelp (we do not use upper-case options)
- Extend the constraint to match on <regex>$(vlink)^</regex>